src/styles/card.scss

```diff
@@ -0,0 +1,121 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: $cardwidth;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+  box-sizing: border-box;
```

Comment:
move this rule to `*` selector

Reply:
`* {
  border-radius: 5px;
  border: 1px solid #f3f3f3;
  background: #fff;
  box-sizing: border-box;
}`
but if I write like this I will apply the frames to all elements on the page, or I don't understand something

Reply:
no, only 
```
* {
  box-sizing: border: box
}
``